showcase/src/app/welcome/pages/reactive-form-errors/demo-reactive-form-errors-page.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, Inject, OnInit } from "@angular/core";
+import { AbstractControl, FormBuilder, FormControl, FormGroup, Validators } from "@angular/forms";
+import { ErrorStateMatcher } from "@angular/material/core";
+import { ParentErrorStateMatcher } from "./parent-error-state-matcher";
+import { PasswordValidator } from "./password-validator";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import {ReferenceLink} from '../../../shared/components/reference-block';
+@Component({
+	selector: "showcase-reactive-forms-with-ngx-form-errors",
+	templateUrl: "./demo-reactive-form-errors-page.component.html",
+	styleUrls: ["./demo-reactive-form-errors-page.component.scss"]
+})
+export class DemoReactiveFormErrorsPageComponent implements OnInit {
+	public collapsed: boolean[] = [false, false, true];
+
+	public referenceList: ReferenceLink[] = [
+		{
+			label: "NGX Form errors repository",
+			url: "https://www.npmjs.com/package/@nationalbankbelgium/ngx-form-errors"
```

Comment:
Question: shouldn't we point to the GitHub page? In my case I prefer to go to the GitHub page of a package because there I can find all the relevant info: releases, changelog and code examples. Normally in NPM you can just find the versions and the README but no changelog so you'll have to go to the GitHub repo for that :s

@SuperItMan @carlo-nomes  What do you guys think?

Reply:
We could add the both ?